cpp/src/arrow/compute/kernels/scalar_arithmetic_test.cc

```diff
@@ -1496,13 +1496,19 @@ TEST_F(TestUnaryArithmeticDecimal, AbsoluteValue) {
       CheckScalar(func, {ArrayFromJSON(ty, R"(["1.00", "-42.15", null])")},
```

Comment:
I think I'm just dumb, but why don't these calls also require explicit construction?

Reply:
There's only one applicable overload (DatumVector). I'm the dumb one here, I added DatumVector below where it wasn't needed. I pushed a fix.

Reply:
Awesome.